src/components/StyleGuide/DesignTokens.stories.mdx

```diff
@@ -0,0 +1,170 @@
+import { withDesign } from "storybook-addon-designs";
+import { getCategory } from "../../utils/componentCategories";
+
+<Meta title={getCategory("Design Tokens")} />
+
+# Design Tokens
+
+| Table of Contents                                                                    |
+| ------------------------------------------------------------------------------------ |
+| 1. [What are Design Tokens?](#what-are-design-tokens)                                |
+| 2. [Why Use Design Tokens?](#why-use-design-tokens)                                  |
+| 3. [How to Use the NYPL Design Tokens](#figma-reference)                             |
+| 4. [Using NYPL Design Tokens in CSS](#using-nypl-design-tokens-in-css)               |
+| 5. [Using NYPL Design Tokens in Javascript](#using-nypl-design-tokens-in-javascript) |
+| 6. [NYPL Design Token Categories](#nypl-design-token-categories)                     |
+
+## What are Design Tokens?
+
+Design tokens are the style values that define a design system. These values can include, but may not be limited to, spacing, color, typography, breakpoints and object styles. Ultimately, these values are used to construct and maintain the components that make up our system.
+
+## Why Use Design Tokens?
+
+_"With design tokens, you can capture low-level values and then use them to create the styles for your product or app. You can maintain a scalable and consistent visual system for UI development."_ – Jina Bolton, Lead Designer for the Salesforce Design System
+
+Design tokens are a single source of truth when it comes to styles. They allow an organization to more easily build multiple applications – web and/or native – that feel and look the same without slowing down the development team.
+
+## How to Use the NYPL Design Tokens
+
+The NYPL design tokens are built into the NYPL Design System (the DS) and are offered in two formats for consumption: CSS variables and javascript data objects. To access either of these formats, the DS v0.25.2 or higher needs to be properly installed in a consuming React application.
+
+## Using NYPL Design Tokens in CSS
+
+If your app uses CSS or SCSS to handle styles, the DS creates standard CSS custom properties (e.g. variables) for consumption in your stylesheets. If the DS is properly installed, the CSS custom properties will be available for you to use. Nothing more needs to be done to your app.
+
+To assist with CSS scope isolation, all of the CSS custom properties created by the DS are prefixed with "nypl." Please see the examples below.
+
+```css
+--nypl-colors-brand-primary
+--nypl-colors-brand-secondary
+--nypl-fontSizes-sm
+--nypl-fontSizes-md
+--nypl-fontSizes-lg
+--nypl-fontSizes-xl
+--nypl-space-xxs
+--nypl-space-xs
+--nypl-space-s
+--nypl-space-m
+--nypl-space-l
+```
+
+The CSS custom properties are defined on the :root pseudo-class and should be implemented using standard CSS custom properties conventions.
+
+```css
+/* property declaration */
```

Comment:
It might be clearer to say something like "Property declaration in the DS" and in the next example "Property usage in your application".

Reply:
Good suggestion.